CHANGELOG.md

```diff
@@ -1,94 +1,119 @@
 # libhoney-js changelog
 
+## [3.1.0] - 2022-04-07
+
+### Enhancements
+
+-   Add support for environments (#244) | [@kentquirk](https://github.com/kentquirk)
+-   ci: add node 17 to test matrix (#195) | [@vreynolds](https://github.com/vreynolds)
+-   empower apply-labels action to apply labels (#205) | [@robbkidd](https://github.com/robbkidd)
+
+### Maintenance
+
+-   gh: add re-triage workflow (#215) | [@vreynolds](https://github.com/vreynolds)
+-   Update dependabot.yml (#212) | [@vreynolds](https://github.com/vreynolds)
+-   Bump @babel/core to 7.16.12 (#230)
+-   Bump @babel/eslint-parser to 7.16.5 (#223)
+-   Bump @babel/preset-env to 7.16.8 (#224)
+-   Bump @rollup/plugin-commonjs to 21.0.1 (#197)
+-   Bump @rollup/plugin-node-resolve to 13.1.3 (#225)
+-   Bump @rollup/plugin-replace to 3.0.1 (#216)
+-   Bump eslint to 8.6.0 (#227)
+-   Bump husky to 7.0.4 (#199)
+-   Bump lint-staged to 12.1.7 (#222)
+-   Bump minimist to 2.5.1 (#220)
+-   Bump superagent from 6.1.0 to 7.0.2 (#226)
+-   Bump vm2 to 3.9.7 (#234)
+
 ## [3.0.0] - 2021-10-18
 
 ### !!! Breaking Changes !!!
 
-- drop node 8 (#188) | [@vreynolds](https://github.com/vreynolds)
+-   drop node 8 (#188) | [@vreynolds](https://github.com/vreynolds)
 
 ### Maintenance
 
-- Change maintenance badge to maintained (#186) | [@JamieDanielson](https://github.com/JamieDanielson)
-- Adds Stalebot (#187) | [@JamieDanielson](https://github.com/JamieDanielson)
-- Bump prettier from 2.4.0 to 2.4.1 (#184)
-- Bump tmpl from 1.0.4 to 1.0.5 (#185)
+-   Change maintenance badge to maintained (#186) | [@JamieDanielson](https://github.com/JamieDanielson)
```

Comment:
hm, is your formatter using the `.editorconfig` in the repo? should we update the config to add the extra space? thinking about the formatting flapping between PRs

Reply:
My formatter is fine -- but `lint-staged`, which is called by the precommit hook, is modifying files AFTER they've been staged. It's ignoring .editorconfig and using its own idea of what a .md file should look like. Apparently that's changed in the most recent version of lint-staged.

Reply:
I've now removed lint-staged (and husky, which was only there to run lint-staged).